INFO:fuefit_excel_runner:Parsed string(@dieselPoints!A1.TABLE(header=True)) as Excel-ref: {'sheet': 'dieselPoints', 'ref': 'A1', 'shape': 'TABLE', 'range_kws': 'header=True', 'pandas_kws': None}
INFO:fuefit_excel_runner:Parsed string(@dieselPoints!A1.TABLE(header=True)) as Excel-ref: {'sheet': 'dieselPoints', 'ref': 'A1', 'shape': 'TABLE', 'range_kws': 'header=True', 'pandas_kws': None}
INFO:fuefit_excel_runner:Parsed string(@Veh2_Petrol_EU5_Class1!A3.table(header=True)) as Excel-ref: {'sheet': 'Veh2_Petrol_EU5_Class1', 'ref': 'A3', 'shape': 'table', 'range_kws': 'header=True', 'pandas_kws': None}
INFO:fuefit_excel_runner:Parsed string(@Veh2_Petrol_EU5_Class1!A3.table(header=True)) as Excel-ref: {'sheet': 'Veh2_Petrol_EU5_Class1', 'ref': 'A3', 'shape': 'table', 'range_kws': 'header=True', 'pandas_kws': None}
Traceback (most recent call last):
  File "D:\Work\fuefit.git\fuefit\excel\fuefit_excel_runner.py", line 221, in run_experiments
    mdl_out = processor.run(mdl_in)
  File "D:\Work\fuefit.git\fuefit\processor.py", line 47, in run
    fitted_coeffs = fit_map(datamodel.resolve_jsonpointer(mdl, '/params/robust_fit'), measured_eng_points)
  File "D:\Work\fuefit.git\fuefit\processor.py", line 141, in fit_map
    (res, _) = curve_fit(fitfunc, df, Y, p0=p0)#, robust=False)
  File "D:\Work\fuefit.git\fuefit\robustfit.py", line 91, in curve_fit
    res = leastsq(f, p0, args=args, full_output=1, **kw)
  File "D:\Apps\WinPython-64bit-3.4.2.1\python-3.4.2.amd64\lib\site-packages\scipy\optimize\minpack.py", line 369, in leastsq
    shape, dtype = _check_func('leastsq', 'func', func, x0, args, n)
  File "D:\Apps\WinPython-64bit-3.4.2.1\python-3.4.2.amd64\lib\site-packages\scipy\optimize\minpack.py", line 20, in _check_func
    res = atleast_1d(thefunc(*((x0[:numinputs],) + args)))
  File "D:\Work\fuefit.git\fuefit\robustfit.py", line 118, in _reweighted_residual_function
    residual    = _weighted_residual_function(params, xdata, ydata, function, s_weights)
TypeError: _weighted_residual_function() missing 1 required positional argument: 'xbounds'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Work\fuefit.git\fuefit\excel\xlwings_code.py", line 7, in <module>
    exp_pairs = mypy.run_experiments(exp_pairs)
  File "D:\Work\fuefit.git\fuefit\excel\fuefit_excel_runner.py", line 225, in run_experiments
    raise Exception('Experiment failed for vehicle(%s): %s' % (veh_id, ex)) from ex 
Exception: Experiment failed for vehicle(veh_1): _weighted_residual_function() missing 1 required positional argument: 'xbounds'
